test/e2e/crud.go

```diff
@@ -23,10 +23,22 @@ import (
 
 	kubeclientset "k8s.io/client-go/kubernetes"
 	"k8s.io/federation/pkg/federatedtypes"
+	"k8s.io/federation/test/common"
 	fedframework "k8s.io/federation/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework"
 )
 
+var _ = framework.KubeDescribe("Federated namespace [Feature:Federation][Experimental]", func() {
```

Comment:
I think there should be a single top-level ``var _ = framework.KubeDescribe(...)`` in a given file with multiple ``Describe`` calls within (with ``test/e2e/namespace.go`` being an example of this structure).

Does this new test belong in ``crud.go`` or would it make more sense to replace the existing test in ``namespace.go``?  If the former, the PR needs to ensure the removal of the existing test.

Reply:
It probably makes more sense to replace the namespace.go test